src/main/resources/db/migration/V20200311165500__set-correct-country-for-states.sql

```diff
@@ -1,11 +1,62 @@
-update states_provinces set
-country_id = (SELECT max(id) FROM countries WHERE name = 'United States of America (the)')
-where name in ('Alabama',
-               'Alaska',
-               'Arizona',
-               'Arkansas',
-               'California',
-               'Colorado',
-               'Connecticut',
-               'Delaware',
-               'Florida');
\ No newline at end of file
+-- The order of the queries is critically important, so please do not change it.
+
+-- Step 1.
+-- find states that related to the wrong country(Austria instead of USA)
+-- 1.1 set USA country for these states in users_data table
+update users_data
+set country_id = (SELECT max(id) FROM countries WHERE name = 'United States of America (the)')
```

Comment:
Aggregate might be expensive operation. Probably `LIMIT 1` is more suitable here, keeping in mind the country name is unique in the table. Could you compare execution plans to find what operation is cheaper?

Reply:
fixed